lib/AST/Builtins.cpp

```diff
@@ -813,6 +828,11 @@ makeMetatype(const T &object, Optional<MetatypeRepresentation> repr = None) {
   return { object, repr };
 }
 
+static BuiltinFunctionBuilder::ParameterPackGenerator
+makeParameterPack(unsigned index = 0) {
```

Comment:
makePackExpansion(). The pattern can be any type, not just a generic parameter type, so it should work exactly like makeMetatype(), etc